### ISSUE TITLE
Missing size-limit 6.0.2 in npm ?

### ISSUE THREAD

**Issue by belgattitude** [ai/size-limit#260]
@size-limit/file is at 6.0.2 but peerDeps of size-limit 6.0.2 seems not on npm (yet) ?

- https://www.npmjs.com/package/size-limit

Might be something on its way, but worth to see if the publishing has failed somewhere.

**Comment by belgattitude** [comment]
@ai still at 6.0.1 https://www.npmjs.com/package/size-limit. Either a very long cache issue, either it's not published.

**Comment by ai** [comment]
Nope. npm denies me to push it right now:

```
npm ERR! 403 403 Forbidden - PUT https://registry.npmjs.org/size-limit - You cannot publish over the previously published versions: 6.0.2.
```
